server/src/main/java/org/apache/iotdb/db/metadata/schemaregion/ISchemaRegion.java

```diff
@@ -293,8 +293,8 @@ ISchemaReader<IDeviceSchemaInfo> getDeviceReader(IShowDevicesPlan showDevicesPla
       throws MetadataException;
 
   /**
-   * The iterated result shall be consumed before calling reader.hasNext() or reader.next(). Its
-   * implementation is based on the reader's process context.
+   * The iterated result shall be consumed before calling reader.hasNextFuture() or reader.next().
```

Comment:
Fix this annotation. Describe these three interfaces usage, isBlocked, hasNext and next.

Reply:
move interfaces usage description to `ISchemaReader`